perceval/runtime/job_group.py

```diff
@@ -222,9 +231,108 @@ def progress(self) -> list:
 
         self._write_job_group_to_disk()  # rewrites the group information on disk
 
-        # replace None with 'not_sent' - more readable
-        status_list = [status if status is not None else "not_sent" for status in status_jobs_in_group]
-        return status_list
+        return status_jobs_in_group
+
+    @staticmethod
+    def _map_job_status_category(status_entry: str) -> JobCategories:
+        # status categories
+        status_success = [RunningStatus.SUCCESS]
+        status_sent = [RunningStatus.RUNNING, RunningStatus.WAITING, RunningStatus.CANCEL_REQUESTED]
+        status_other = [RunningStatus.ERROR, RunningStatus.CANCELED, RunningStatus.SUSPENDED, RunningStatus.UNKNOWN]
+
+        if status_entry is None:
+            return JobCategories.UNFIN_NOT_SENT
+
+        elif RunningStatus[status_entry] in status_sent:
+            return JobCategories.UNFIN_SENT
+
+        elif RunningStatus[status_entry] in status_success:
+            return JobCategories.FIN_SUCCESS
+
+        elif RunningStatus[status_entry] in status_other:
+            return JobCategories.FIN_OTHER
+        else:
+            raise ValueError(f"Unspecified status of job in group with value {status_entry}. Cannot categorize")
+
+    def progress(self) -> dict:
```

Comment:
The signature has changed --> this should be written somewhere

Reply:
The method does return a python dict, I do not understand which signature has changed.